colors/gruvbox.vim

```diff
@@ -1592,6 +1592,16 @@ hi! link rustDefault GruvboxAqua
 
 " }}}
 
+" Ocaml: {{{
+
+hi! link ocamlOperator GruvBoxAqua
+hi! link ocamlKeyChar GruvBoxAqua
+hi! link ocamlArrow GruvboxAqua
+hi! link ocamlInfixOpKeyword GruvboxAqua
+hi! link ocamlConstructor GruvboxYellow
```

Comment:
Something like this would take the colours from the already-defined categories, making the code look more consistent with other languages. (The groups were taken from the equivalents in [nvim-tree-sitter](https://github.com/nvim-treesitter/nvim-treesitter/blob/7e3c4f808940a981a034709163724eb5a7546e2d/plugin/nvim-treesitter.vim))

```suggest
hi! link ocamlOperator Operator
hi! link ocamlKeyChar Delimiter
hi! link ocamlArrow Delimiter
hi! link ocamlInfixOpKeyword Keyword
hi! link ocamlConstructor Special
```

Reply:
How the code looks like with this change:
![Screenshot_20201003_164527](https://user-images.githubusercontent.com/578125/94995712-0e6b1a00-0598-11eb-9580-f4bc6b05455f.png)

Compare with tree-sitter highlighting:
![Screenshot_20201003_164057](https://user-images.githubusercontent.com/578125/94995721-1e82f980-0598-11eb-8fd5-8a9c482ebdf4.png)